docker-compose.yml

```diff
@@ -66,11 +66,15 @@ services:
     healthcheck:
       test: "curl --silent --fail localhost:9200/_cluster/health || exit 1"
 
-  rummager:
+  rummager: &rummager
     build: apps/rummager
     command: bundle exec unicorn -p 3009
     depends_on:
       - diet-error-handler
+      - rummager-worker-1
+      - rummager-worker-2
+      - rummager-worker-3
+      - rummager-worker-4
```

Comment:
Just a small suggestion that it might be nicer to name these a bit more descriptive could have:
```
- rummager-worker
- rummager-listener-publishing-queue
- rummager-listener-insert-data
- rummager-listener-bulk-insert-data
```

what do you think?

Reply:
We were thinking that might be better yeah